guide/popular-topics/miscellaneous-examples.md

```diff
@@ -46,7 +46,7 @@ client.on('message', message => {
 	if (message.content === '!play') {
 		if (message.channel.type !== 'text') return;
 
-		const { voiceChannel } = message.member;
+		const { voiceChannel } = message.member.voice;
```

Comment:
```suggestion
		const voiceChannel = message.member.voice.channel;
```
or `const { channel: voiceChannel }` if you wanted to maintain the destructing assignment

Reply:
Updated